### ISSUE TITLE
The issuer certificate could not be found

### ISSUE THREAD

**Issue by Grynde** [sqlitebrowser/sqlitebrowser#3331]
Details for the issue
--------------------

#### What did you do?
Open app.

#### What did you expect to see?

No error.
#### What did you see instead?

"Error opening remote file at https://download.sqlitebrowser.org/currentrelease. The issuer certificate could not be found"

I tried both 32 and 64.... same error same time. Just on open.

If I navigate to that link i get:

3.12.2
https://sqlitebrowser.org/blog/version-3-12-2-released

Useful extra information
-------------------------
> DB4S v3.12.2 [built for x86_64-little_endian-llp64] on Windows 10 (10.0) (winnt/10.0.19044) [x86_64]
> using SQLite Version 3.35.5
<img width="251" alt="Screenshot 2023-04-26 103642" src="https://user-images.githubusercontent.com/105419112/234518947-b79d1a62-d3b5-48ad-a0cf-e508a035d5e6.png">

> and Qt 5.12.8

**Comment by justinclift** [comment]
Oh, that's weird.  I wonder if it's related to our (ongoing) server migration.

@chrisjlocke is this you doing DNS stuff? :smile:

**Comment by justinclift** [comment]
@Grynde Does it still happen if you try starting the application again?

I've just tried it out here with a recent nightly build in a Windows VM, and for me there's no error happening.

If it's still happening for you though, that means something weird is going on.

**Comment by Grynde** [comment]
Yes. It has happened every single time I have started the app. But after pressing OK it was working OK.  

I have installed 32 and 64 alternatively af few times.

So for the moment I have just disabled "Automatic Updates" in Preferences. 

As far as I know I have a conventional network setup, but in this case I was using my company computer. Maybe I can check later trying to use my personal computer.

**Comment by lucydodo** [comment]
A minor question, when you access `https://download.sqlitebrowser.org/currentrelease` in a browser,
is an HTTPS connection established and the certificate verified?

**Comment by justinclift** [comment]
Interesting.  Just installed our 3.12.2 release on the same VM, and again it's not showing any error.

So, my initial guess here is that there's *something* about your work computer that's blocking part of things.

Perhaps there's a security suite installed and it's doing something funky?

Having seen the quality of work-mandated security software before (its generally pretty bad), that wouldn't super surprise me. :wink:

**Comment by Grynde** [comment]
for lucydodo:

Yes. I get :

**Comment by lucydodo** [comment]
Interesting. I thought it might be be the security configuration on the corporate network.
but if the browser is establishing an HTTPS connection, I think there's something else to think about. 🤔

**Comment by lucydodo** [comment]
When referencing issue #2851, there have been report of DB4S running fine after accessing that URI with a browser.
Seems worth a shot if you haven't tried it. :smile:

**Comment by Grynde** [comment]
Just to confirm that in my personal computer it works fine.  So probably it is some "security" feature installed in the corporate computer (I am working from home and not using VPN).

**Comment by lucydodo** [comment]
Thanks for checking it out. It looks like your company security policy is the cause of the problem.
Would it be okay to close to issue? cc @justinclift

**Comment by Grynde** [comment]
I have accessed with CR and Edge but while the link is accesible, it has not changed the status on app open. In any case, it seems it is something specific to this computer, so you can close the issue.

I will let you  know if something else pops up while browsing the DB.

**Comment by justinclift** [comment]
Yeah, there's nothing we can do on our end to help fix it.  It's probably something the company IT people would need to look into.

For a simple update check, that's unlikely to be worth the effort. :wink:

**Comment by chrisjlocke** [comment]
Why would the 'current release' text file have a link to the blog post?
Surely it should link to the download link for the application? (v3.12.2)
